atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/SweepQueueDeleter.java

```diff
@@ -48,19 +55,32 @@
     public void sweep(Collection<WriteInfo> writes, Sweeper sweeper) {
         Map<TableReference, Map<Cell, Long>> maxTimestampByCell = writesPerTable(writes, sweeper);
         for (Map.Entry<TableReference, Map<Cell, Long>> entry : maxTimestampByCell.entrySet()) {
-            Iterables.partition(entry.getValue().keySet(), SweepQueueUtils.BATCH_SIZE_KVS)
-                    .forEach(cells -> {
-                        Map<Cell, Long> maxTimestampByCellPartition = cells.stream()
-                                .collect(Collectors.toMap(Function.identity(), entry.getValue()::get));
-                        follower.run(entry.getKey(), maxTimestampByCellPartition.keySet());
-                        if (sweeper.shouldAddSentinels()) {
-                            kvs.addGarbageCollectionSentinelValues(entry.getKey(),
-                                    maxTimestampByCellPartition.keySet());
-                            kvs.deleteAllTimestamps(entry.getKey(), maxTimestampByCellPartition, false);
-                        } else {
-                            kvs.deleteAllTimestamps(entry.getKey(), maxTimestampByCellPartition, true);
-                        }
-                    });
+            try {
+                Iterables.partition(entry.getValue().keySet(), SweepQueueUtils.BATCH_SIZE_KVS)
+                        .forEach(cells -> {
+                            Map<Cell, Long> maxTimestampByCellPartition = cells.stream()
+                                    .collect(Collectors.toMap(Function.identity(), entry.getValue()::get));
+                            follower.run(entry.getKey(), maxTimestampByCellPartition.keySet());
+                            if (sweeper.shouldAddSentinels()) {
+                                kvs.addGarbageCollectionSentinelValues(entry.getKey(),
+                                        maxTimestampByCellPartition.keySet());
+                                kvs.deleteAllTimestamps(entry.getKey(), maxTimestampByCellPartition, false);
+                            } else {
+                                kvs.deleteAllTimestamps(entry.getKey(), maxTimestampByCellPartition, true);
+                            }
+                        });
+            } catch (IllegalArgumentException e) {
+                // hack because InMemoryKVS doesn't throw a TableMappingNotFoundException...
+                if (e.getCause() instanceof TableMappingNotFoundException
+                        || e.getMessage().endsWith("does not exist")) {
+                    log.debug("Could not resolve full name for table reference {}, "
```

Comment:
Could log at a higher priority here.

I feel slightly nervous about just dropping the items, but we don't have a concept of a Dead Letter Queue for storing failed messages.  Is it worth logging the cells here?  Am I being over cautious here?